PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/CameraScript/RootCamera.rbxmx

```diff
@@ -68,7 +75,17 @@ local HEAD_OFFSET = Vector3.new(0, 1.5, 0)
 -- Reset the camera look vector when the camera is enabled for the first time
 local SetCameraOnSpawn = true
 
-
+local function EvaluatePitchLimits()
+	if not VRAPIExists then return end
+	if UserInputService.VREnabled then 
+		PitchMin = MIN_Y
+		PitchMax = MAX_Y
+	else
+		PitchMin = VR_MIN_Y
+		PitchMax = VR_MAX_Y
+	end
+end
+UserInputService.Changed:connect(EvaluatePitchLimits)
```

Comment:
This is a good idea to move the logic to an event.
We should put the connection logic in the `this:ConnectInputEvents` and `this:DisconnectInputEvents` functions so that it cleans itself up when not in use.

Also, I think we call `EvaluatePitchLimits()` when we connect to the event as well.